intelmq/bots/experts/rdap/expert.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import json
+from urllib.parse import urlparse
+
+from intelmq.lib.bot import Bot
+from intelmq.lib.cache import Cache
+from intelmq.lib.utils import create_request_session
+from intelmq.lib.exceptions import MissingDependencyError
+
+try:
+    import requests
+except ImportError:
+    requests = None
+
+
+class RDAPExpertBot(Bot):
+    """ Get RDAP data"""
+    rdap_order = ['abuse', 'technical', 'administrative', 'registrant', 'registrar']
+    redis_cache_db: int = 5
```

Comment:
That's the same cache database as the deduplicator uses by default. Can we choose some database which is not in use? E.g. 8